examples/react/eslint.config.js

```diff
@@ -0,0 +1,26 @@
+import js from '@eslint/js';
```

Comment:
Do we need this in example? We already have a `eslint.config.js` which handles the linting in this repo. If you need more react-related rules, could you please add it the the root `eslint.config.js` instead of creating a new one?